pkg/security/admission/admission.go

```diff
@@ -216,40 +240,51 @@ func (c *constraint) createProvidersFromConstraints(ns string, sccs []*kapi.Secu
 				continue
 			}
 
+			// Make a copy of the constraint so we don't mutate the store's cache
+			var constraintCopy kapi.SecurityContextConstraints = *constraint
+			constraint = &constraintCopy
```

Comment:
Why not make this a deep copy to avoid these shenanigans later on: https://github.com/openshift/origin/pull/5079/files#diff-eb43ba779f411e3ee6b772cceee9810dR262